src/20_Components/amp-form.html

```diff
@@ -71,5 +74,23 @@
         </template>
     </div>
   </form>
+
+  <!--
+    Use the `amp-form-submit-success` class to hide inputs after a successfull submission.
```

Comment:
You could embed the corresponding css code:

This hides all form input fields after successful form submission:
```
form.amp-form-submit-success.hide-inputs > input {
    display: none
}
```